src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -614,8 +614,8 @@ public void updateTransportInfo(
      * {@inheritDoc}
      */
     @Override
-    public void updateSourcesInfo(MediaSSRCMap ssrcs,
-                                  MediaSSRCGroupMap ssrcGroups,
+    public void updateSourcesInfo(MediaSourceMap ssrcs,
```

Comment:
yes you're right, i did this in most places but looks like i missed this one.  will fix it.